### DISCUSSION TITLE
Love this app!

### DISCUSSION THREAD

**Discussion by fred919** [cyanfish/naps2#339] [General]
I didn't know where to post a review so I'm posting here instead.

I found this app many yrs ago after Acrobat, which I'd also used for scanning to PDF, switched to subscription. Even after licensing an alternative PDF editor I remain hooked on this app. It works really well and keeps improving. Adding the manual duplex options were huge for me - although I have some trouble remembering which one to use. The profile toolbar was another great improvement for me. I'll keep donating from time to time. Please keep up the great work!

...Fred

**Comment by infoforq** [comment]
I love this software too. This solution solves many of my problems with scanner and document handling. But I would like to make a suggestion: put the "duplicate" option when we right-click on one of the scanned pages
